library/SimplePie.php

```diff
@@ -650,6 +650,13 @@ class SimplePie
 	 */
 	public $strip_htmltags = array('base', 'blink', 'body', 'doctype', 'embed', 'font', 'form', 'frame', 'frameset', 'html', 'iframe', 'input', 'marquee', 'meta', 'noscript', 'object', 'param', 'script', 'style');
 
+	/**
+	 * @var array Stores the default tags to be stripped by rename_attributes().
```

Comment:
Maybe use `string[]` as a more detailed type annotation.

Reply:
I want to keep it similar to:

https://github.com/simplepie/simplepie/blob/1aec297145a150f627e6ed82b8b084e0c933def8/library/SimplePie.php#L637-L644